blueocean-dashboard/src/main/js/components/Step.jsx

```diff
@@ -160,13 +168,14 @@ export default class Node extends Component {
     }
 }
 
-const { object, func, string, bool } = PropTypes;
+const { object, func, string, bool, shape } = PropTypes;
 Node.propTypes = {
     node: object.isRequired,
     followAlong: bool,
     logs: object,
     location: object,
     fetchLog: func,
     nodesBaseUrl: string,
+    router: shape,
```

Comment:
Yeah, monday I updated the eslint rules and in the new ones you cannot use "object, array and any" as proptypes anymore, so I guess that was still in the back of my head